modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/FlowUtil.kt

```diff
@@ -0,0 +1,24 @@
+package au.com.shiftyjelly.pocketcasts.utils
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.combine as kotlinCombine
+
+public fun <T1, T2, T3, T4, T5, T6, R> combine(
```

Comment:
Can we reuse existing flow extension: `fun <T1, T2, T3, T4, T5, T6, R> combine6` instead of creating a new one?

Reply:
Oh didn't notice we have one. Would you mind if I removed `combine6()` and keep this one instead? I'd prefer to follow Kotlin's naming.

Reply:
yep, you can remove combine6

Reply:
Fixed in 21a3f2eb08c0162bdfb455b78e667877864d7f6e